backend/server.js

```diff
@@ -34,6 +35,17 @@ app.use(express.static(path.join(__dirname, 'public')));
 app.use('/', indexRouter);
 app.use('/users', usersRouter);
 
+require('dotenv').config(); // sensitive information
+
+//connect to mongo cluster 
+const uri = process.env.ATLAS_URI;
+mongoose.connect(uri, { useNewUrlParser: true, useUnifiedTopology: true });
+
+const connection = mongoose.connection;
+connection.once('open', () => {
+    console.log("Connection to Mongo Cluster established!");
+});
+
 // catch 404 and forward to error handler
```

Comment:
Instead of putting an inline comment here, were should consider adding error logs to document this.